### ISSUE TITLE
UnpickleStateDict on BatchNorm2d error

### ISSUE THREAD

**Issue by dongfangzhizhu** [shaltielshmid/TorchSharp.PyBridge#13]
On BatchNorm2d ,the track_running_stats is default value true,by UnpickleStateDict function get num_batches_tracked is [1], type = Int64,this  raise exception.But if convert pytorch_model.bin to pytorch_model.safetensors format,get num_batches_tracked is [], type = Int64,it' load right.

**Comment by shaltielshmid** [comment]
Thank you for pointing this out! I'll look into it.

**Comment by shaltielshmid** [comment]
Should be fixed with this commit: https://github.com/shaltielshmid/TorchSharp.PyBridge/commit/459d9c49d1451f668c55d88b273e6ccc24b85bdd

I added a release with this fix (version 1.3.2), and it should be available on NuGet shortly.
